samples/nginx/propagationpolicy.yaml

```diff
@@ -11,3 +11,4 @@ spec:
     clusterAffinity:
       clusterNames:
         - member1
+        - member2
```

Comment:
This willl change the output above:
```
kubectl get deployment
NAME    READY   UP-TO-DATE   AVAILABLE   AGE
nginx   1/1     1            1           43s
```

`1/1` --> `2/1`. That will confuse people that not familiar with Karmada.

Reply:
My suggestion is to update the PropagationPolicy to `divide` replicas.